package/kedro_viz/server.py

```diff
@@ -57,8 +57,13 @@
     from kedro.framework.cli import get_project_context
     from kedro.framework.cli.utils import KedroCliError
 else:
-    from kedro.cli import get_project_context  # pragma: no cover
-    from kedro.cli.utils import KedroCliError  # pragma: no cover
+
+    from kedro.cli import (  # pylint: disable=no-name-in-module,import-error # pragma: no cover
+        get_project_context,
+    )
+    from kedro.cli.utils import (  # pylint: disable=no-name-in-module,import-error # pragma: no cover
+        KedroCliError,
+    )
```

Comment:
if you have time, I'd fix the comments and put them on the line above to stop black breaking it into multiple lines (here and in other places)

not critical though :)